index.js

```diff
@@ -3,9 +3,9 @@ const childProcess = require('child_process');
 const execa = require('execa');
 const mem = require('mem');
 
-function getEnvVar() {
+function getEnvVar(options) {
```

Comment:
If you want `root` when your program is run with sudo.
AFAIK there is no 'standard' as to how a program should behave when run with sudo so it's really up to the program author.